utils/audioFormats.ts

```diff
@@ -0,0 +1,56 @@
+import { browserDetector } from '~/plugins/browserDetection';
+
+/**
+ *
+ *
+ *
+ * @param {string} format
+ * @returns
+ */
+export function getSupportedAudioCodecs(format: string): boolean {
+  let typeString;
+
+  if (format === 'flac') {
+    if (browserDetector.isTv()) {
+      return true;
+    }
+  } else if (format === 'wma') {
+    if (browserDetector.isTizen()) {
+      return true;
+    }
+  } else if (format === 'asf') {
+    if (browserDetector.isTv()) {
+      return true;
+    }
+  } else if (format === 'opus') {
+    if (!browserDetector.isWebOS()) {
+      typeString = 'audio/ogg; codecs="opus"';
+      return !!document
```

Comment:
```suggestion
      return document

Reply:
This is there to silence TypeScript complaining that I'm not returning a boolean :p

Though we could probably change the return type to string, but it feels a little bit weird to have a flag be a string...

Reply:
Oo ok, makes sense